OloEngine/src/OloEngine/Renderer/Debug/CommandPacketDebugger.cpp

```
@@ -1,834 +1,1104 @@
+            case CommandType::SetCulling:
+            case CommandType::SetPolygonMode:
+                return ImVec4(0.3f, 0.3f, 0.8f, 1.0f); // Blue
 
             default:
-                return ImVec4(0.8f, 0.5f, 0.3f, 1.0f); // Orange for other commands
+                return ImVec4(0.8f, 0.5f, 0.3f, 1.0f); // Orange
         }
     }
-    std::string CommandPacketDebugger::GetPacketTypeString(const CommandPacket* packet) const
+
+    const char* CommandPacketDebugger::GetCommandTypeString(CommandType type)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)